guardrails/guardrail_globals.py

```diff
@@ -14,11 +16,13 @@ def __init__(self):
         self.test_folder = None
         self.pytest = None
         self.report_folder = None
+        self.jscpd_root = None
         self.cyclo_exclude = None
         self.python = None
         self.pylintrc = None
         self.covrc = None
         self.dup_token = 50
+        self.min_deadcode_confidence = 60
```

Comment:
This was originally set to 100 isn't it? Did we change it back to 60?

Reply:
100 in most of the case pylint also reflects. hence keeping at 60. So team can play with the config val